ChangeLog.d/remove-via-padlock-support.txt

```diff
@@ -0,0 +1,5 @@
+Removals
+   * Drop support for VIA Padlock. Removes MBEDTLS_PADLOCK_C and
+     associated code.
+   * Note that it is still possible to use VIA Padlock through a
+     PSA accelerator driver that is not part of Mbed TLS. Fixes #5903.
```

Comment:
And actually the problem with the ChangeLog entry that the CI is complaining about is the lack of trailing `\n`, even though GitHub isn't showing it as an issue here (normally it puts an indicator symbol at the end of the file)